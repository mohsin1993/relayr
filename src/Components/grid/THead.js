import React from 'react';

const THead = () => (
  <thead>
    <tr>
      <th>Name</th>
      <th>Unit</th>
      <th>Value</th>
      <th>Timestamp</th>
      <th>Active</th>
    </tr>
  </thead>
);

export default THead;
