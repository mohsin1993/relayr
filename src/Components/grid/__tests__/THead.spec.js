import React from 'react';
import snapshot from 'check-snapshot';
import THead from '../THead';

describe('THead', () => {
  it('It renders THead correctly', () => {
    snapshot(
      <table>
        <THead />
      </table>
    );
  });
});
