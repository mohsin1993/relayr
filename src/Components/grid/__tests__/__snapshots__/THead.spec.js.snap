// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`THead It renders THead correctly 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Name
      </th>
      <th>
        Unit
      </th>
      <th>
        Value
      </th>
      <th>
        Timestamp
      </th>
      <th>
        Active
      </th>
    </tr>
  </thead>
</table>
`;
